srv/GetSpawnables.srv

```diff
@@ -0,0 +1,9 @@
+# Return a list of resources which are valid as SpawnEntity uri fields (e.g. visible to or registered in simulator).
+# This interface is an optional extension and might not be implemented by your simulator, check the status_message.
+
```

Comment:
Going off my suggestion to have optional entity categories, this could be a nice place to use it in the service request.